cmd/livepeer/starter/starter.go

```diff
@@ -908,16 +781,36 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 
 		if *cfg.Orchestrator {
 			// Set price per pixel base info
+			pixelsPerUnit, ok := new(big.Rat).SetString(*cfg.PixelsPerUnit)
+			if !ok || !pixelsPerUnit.IsInt() {
+				panic(fmt.Errorf("-pixelsPerUnit must be a valid integer, provided %v", *cfg.PixelsPerUnit))
+			}
+			if pixelsPerUnit.Sign() <= 0 {
+				// Can't divide by 0
+				panic(fmt.Errorf("-pixelsPerUnit must be > 0, provided %v", *cfg.PixelsPerUnit))
+			}
 			if cfg.PricePerUnit == nil && !*cfg.AIWorker {
 				// Prevent orchestrators from unknowingly providing free transcoding
 				panic(fmt.Errorf("-pricePerUnit must be set"))
 			} else if cfg.PricePerUnit != nil {
-				if *cfg.PricePerUnit < 0 {
-					panic(fmt.Errorf("-pricePerUnit must be >= 0, provided %d", *cfg.PricePerUnit))
+				pricePerUnit, currency, err := parsePricePerUnit(*cfg.PricePerUnit)
```

Comment:
@victorges please duble check if I handled this conflict correctly so that it also works as expected on the transcoding side. Thanks in advance 🙏🏻.

Reply:
Made a couple comments about this on the specific commit. TLDR I'd say the currency should go in a separate field in the config, like on the per gateway prices, and not on the same field as a string.

ref: https://github.com/livepeer/go-livepeer/pull/3103#discussion_r1700906838